### DISCUSSION TITLE
Cannot explore URL: file:/home/runner/.m2/repository/commons-logging/commons-logging/1.2/commons-logging-1.2.jar: error in opening zip file

### DISCUSSION THREAD

**Discussion by prernasri1992** [community/community#127862] [Actions]
### Select Topic Area

General

### Body

I am trying to use Git action for CICD for my MuleSoft application. However I am facing below error.
_**Cannot explore URL: file:/home/runner/.m2/repository/commons-logging/commons-logging/1.2/commons-logging-1.2.jar: error in opening zip file**_
<img width="1478" alt="image" src="https://github.com/community/community/assets/92391481/768b0b69-eb03-4b6d-8acf-439d9fe56f3e">

**Comment by vishal11u** [comment]
Your JDK path was not correct add in environment system in your device
